m/mysql-connector-j/mysql-connector-j_ubi_9.3.sh

```diff
@@ -0,0 +1,110 @@
+wget https://search.maven.org/remotecontent?filepath=org/junit/jupiter/junit-jupiter-api/5.10.2/junit-jupiter-api-5.10.2.jar -O junit-jupiter-api-5.10.2.jar
+wget https://search.maven.org/remotecontent?filepath=org/junit/jupiter/junit-jupiter-engine/5.10.2/junit-jupiter-engine-5.10.2.jar -O junit-jupiter-engine-5.10.2.jar
+
+wget https://search.maven.org/remotecontent?filepath=org/junit/platform/junit-platform-commons/1.10.2/junit-platform-commons-1.10.2.jar -O junit-platform-commons-1.10.2.jar
+wget https://search.maven.org/remotecontent?filepath=org/junit/platform/junit-platform-engine/1.10.2/junit-platform-engine-1.10.2.jar -O junit-platform-engine-1.10.2.jar
+wget https://search.maven.org/remotecontent?filepath=org/junit/platform/junit-platform-launcher/1.10.2/junit-platform-launcher-1.10.2.jar -O junit-platform-launcher-1.10.2.jar
+
+wget https://search.maven.org/remotecontent?filepath=org/apiguardian/apiguardian-api/1.1.2/apiguardian-api-1.1.2.jar -O apiguardian-api-1.1.2.jar
+wget https://search.maven.org/remotecontent?filepath=org/opentest4j/opentest4j/1.3.0/opentest4j-1.3.0.jar -O opentest4j-1.3.0.jar
+
+wget https://search.maven.org/remotecontent?filepath=org/javassist/javassist/3.30.2-GA/javassist-3.30.2-GA.jar -O javassist-3.30.2-GA.jar
+wget https://search.maven.org/remotecontent?filepath=com/google/protobuf/protobuf-java/4.27.2/protobuf-java-4.27.2.jar -O protobuf-java-4.27.2.jar
+wget https://search.maven.org/remotecontent?filepath=com/mchange/c3p0/0.10.1/c3p0-0.10.1.jar -O c3p0-0.10.1.jar
+wget https://search.maven.org/remotecontent?filepath=org/slf4j/slf4j-api/2.0.13/slf4j-api-2.0.13.jar -O slf4j-api-2.0.13.jar
+wget https://search.maven.org/remotecontent?filepath=org/hamcrest/hamcrest/2.2/hamcrest-2.2.jar -O hamcrest-2.2.jar
+wget https://search.maven.org/remotecontent?filepath=io/opentelemetry/opentelemetry-api/1.38.0/opentelemetry-api-1.38.0.jar -O opentelemetry-api-1.38.0.jar
+wget https://search.maven.org/remotecontent?filepath=io/opentelemetry/opentelemetry-context/1.38.0/opentelemetry-context-1.38.0.jar -O opentelemetry-context-1.38.0.jar
+wget https://search.maven.org/remotecontent?filepath=org/opentest4j/opentest4j/1.3.0/opentest4j-1.3.0.jar -O opentest4j-1.3.0.jar
+wget https://search.maven.org/remotecontent?filepath=com/oracle/oci/sdk/oci-java-sdk-common/3.41.2/oci-java-sdk-common-3.41.2.jar -O oci-java-sdk-common-3.41.2.jar
+
+cd $WORKDIR
+
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+
+#Check if package exists
+if [ -d "$PACKAGE_NAME" ] ; then
+  rm -rf $PACKAGE_NAME
+  echo "$PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Removed existing package if any"  
+ 
+fi
+
+if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+    	echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+		echo "$PACKAGE_URL $PACKAGE_NAME"
+        echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
+    	exit 0
```

Comment:
Exit 0 only at the end of the script and for success scenario.

Reply:
added fix